src/orion/core/io/resolve_config.py

```diff
@@ -181,8 +182,8 @@ def fetch_metadata(cmdargs):
             user_script = abs_user_script
 
     if user_script and not os.path.exists(user_script):
-        raise Exception(
-            'The path {} specified for the script does not exist'.format(user_script))
+        raise OSError(
+            errno.ENOENT, "The path {} specified for the script does not exist".format(user_script))
```

Comment:
A more standard use of `OSError` would be without the format.
```python
OSError(errno.ENOENT, "The path specified for the script does not exist", user_script)
```

Giving:

```text
FileNotFoundError: [Errno 2] The path specified for the script does not exist: 'some_path'
```

Reply:
No, it won't print the path. The output will be : `FileNotFoundError(2, 'The path specified for the script does not exist')`

Reply:
It depends, the `raise` will do, the `__repr__` of the error won't.

```python
>>> raise OSError(errno.ENOENT, "The path specified for the script does not exist", 'some_path')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] The path specified for the script does not exist: 'some_path'

>>> OSError(errno.ENOENT, "The path specified for the script does not exist", 'some_path')
FileNotFoundError(2, 'The path specified for the script does not exist')
```

Reply:
Got it. Thanks!